ocs_ci/ocs/node.py

```diff
@@ -1233,17 +1233,27 @@ def node_replacement_verification_steps_ceph_side(
 
     ct_pod = pod.get_ceph_tools_pod()
     ceph_osd_status = ct_pod.exec_ceph_cmd(ceph_cmd="ceph osd status")
-    if new_osd_node_name not in ceph_osd_status:
```

Comment:
I don't understand this condition and the change in line 1209 
if no `new_osd_node_name` was passed it will be None by default and the func will return false by default 
so why is this `=None` in row 1209 needed?

Reply:
so I kept reading to the end of the if/if/else and I thing that the logic of the code is not right 
I thing that the if part from line 1236 to line 1238 should inside the if condition in row 1240